cleaner.go

```diff
@@ -253,6 +261,90 @@ func (c *Cleaner) removeFinalizers(gvk schema.GroupVersionKind, ns string) error
 	return nil
 }
 
+func (c *Cleaner) getFunction(sbu unstructured.Unstructured) (*serverless.Function, error) {
+	if sbu.Object == nil {
+		return nil, fmt.Errorf("SBU %v/%v has no body object", sbu.GetNamespace(), sbu.GetName())
+	}
+	kind, found, err := unstructured.NestedString(sbu.Object, "spec", "usedBy", "kind")
+	if !found || err != nil {
+		return nil, fmt.Errorf("SBU %v/%v missing spec.usedBy.kind", sbu.GetNamespace(), sbu.GetName())
+	}
+	if kind != "serverless-function" {
+		return nil, fmt.Errorf("SBU %v/%v unsupported spec.usedBy.kind %v", sbu.GetNamespace(), sbu.GetName(), kind)
```

Comment:
by default, in fresh Kyma, we have 2 usagekinds defined - for functions and deployments.

Reply:
We have to decide what to do is we have another custom usage kind, let's sync with @PK85